VKProject/Src/Tests/Tests.cs

```diff
@@ -0,0 +1,46 @@
+using FluentAssertions;
+using NUnit.Framework;
+using VKProject.Constants;
+using VKProject.Models;
+using VKProject.Pages;
+using VKProject.Steps;
+using VKProject.Steps.ApiSteps;
+using VKProject.Tests.BaseUiTest;
+using VKProject.Utils;
+
+namespace VKProject.Tests;
+
+public class Tests : BaseTest
+{
+    private Post _post;
+    private Comment _comment;
+    private User _user;
+
+    [SetUp]
+    public void SetUp()
+    {
+        _post = TestDataGeneratorService.GetFakePost();
+        _comment = TestDataGeneratorService.GetFakeComment();
+        _user = DataBaseReader.GetUser(1);
+    }
+
+    [Test]
+    public void TC1()
+    {
+        LoginSteps.Login(1);
+        NewsPage.MoveToMyPage();
+        var postId = PostApiHelper.CreatePost(1, _post);
+        MyPage.GetCreatorNameFromPost(postId).Should().Be(_user.userId);
+        MyPage.GetTextFromPost(postId).Should().Be(_post.Text);
+        _post = TestDataGeneratorService.GetFakePost();
```

Comment:
you are getting _post in spite of SetUp getting?

Reply:
In the task, I need to edit the post and add new text to it, for this I call the faker again to change the text